tests/TestClassRedirectIfOldUrl.php

```diff
@@ -75,6 +75,7 @@ public function testGetUri()
 	 */
 	public function testRedirectDetailViewSameUrl()
 	{
+		add_option('onoffice-detail-view-showTitleUrl', true);
```

Comment:
Both with and without that option should be tested

Reply:
Hi @jayay,
In this case I cannot assert for `wp_redirect`. I think WordPress does not support this case yet

Reply:
Yeah, a wrapper class for `wp_redirect` would be needed, which in turn could be mocked for the test.